metadata-ingestion/src/datahub/ingestion/source/looker/lookml_sql_parser.py

```diff
@@ -54,30 +54,42 @@ def _update_fields(fields: List[ViewField], spr: SqlParsingResult) -> List[ViewF
         view_field_map[field.name] = field
 
     for cll in column_lineages:
-        upstream_fields = [
-            (LookerFieldName(column_ref)).name() for column_ref in cll.upstreams
-        ]
-
         if view_field_map.get(cll.downstream.column) is None:
             logger.debug(
                 f"column {cll.downstream.column} is not present in view defined using sql"
                 f"measure/dimension/dimension_group"
             )
             continue
 
-        view_field_map[cll.downstream.column].upstream_fields = upstream_fields
+        view_field_map[cll.downstream.column].upstream_fields = cll.upstreams
 
     return fields
 
 
+def get_qt_name(urn: str) -> str:
```

Comment:
what is qt

Reply:
It is get_qualified_table_name. I am going to remove `LookerFieldName`, `get_qt_name` and `get_qt_names_from_spr` as they are not required now